bindings/core/src/method/wallet.rs

```diff
@@ -25,13 +43,6 @@ use crate::OmittedDebug;
 #[serde(tag = "name", content = "data", rename_all = "camelCase")]
 #[non_exhaustive]
 pub enum WalletMethod {
```

Comment:
We could take the opportunity to sort it but we can do that later

Reply:
Already thought you will comment on that :trollface: But yes I would wait a bit until we decided what we want to keep/remove